static/app/views/performance/newTraceDetails/index.tsx

```diff
@@ -535,13 +532,9 @@ function TraceViewContent(props: TraceViewContentProps) {
     [setTracePreferences]
   );
 
-  const resizeAnimationTimeoutRef = useRef<{id: number} | null>(null);
   const onDrawerResize = useCallback(
     (size: number) => {
-      if (resizeAnimationTimeoutRef.current !== null) {
-        cancelAnimationTimeout(resizeAnimationTimeoutRef.current);
-      }
-      resizeAnimationTimeoutRef.current = requestAnimationTimeout(() => {
+      startTransition(() => {
```

Comment:
marks this state update as low priority https://react.dev/reference/react/startTransition

Reply:
this function doesn't seem to exist on 17